### DISCUSSION TITLE
Cloning EG8010H data to DFP-34X-2C2

### DISCUSSION THREAD

**Discussion by dadanielat** [Anime4000/RTL960x#63] [Q&A]
Dear contributors,
I just ordered a DFP-34X-2C2 and want to replace my provider's EG8010H ONT. They are using a MA5800-X2 OLT.
Can you please help me to match the info I got from EG8010H below to initial settings suitable for the DFP-34X-2C2? (I've slightly modified the data for privacy purposes):

S/N: 2102713WCR1RM5003232
GPON ID: 487259430108F0A6
MAC Address: 60:7e:cd:5b:07:2a
DHCP Circuit ID: POP1-MA5800-X2 xpon 0/1/0/4:63.28

Thank you very much for your great effort!
best regards
Daniel

**Comment by Anime4000** [comment]
your Fiber auth is PLOAM or LOID ?

please get more info about your current ONU like this

![example](https://github.com/Anime4000/RTL960x/raw/main/Docs/Images/HG8240H5_example.png)

I have generate new key for your mac address
```
flash set ELAN_MAC_ADDR 607ECD5B072A
flash set MAC_KEY 506169001ccc07867cf33300053306a2
```

**Reply by dadanielat** [reply]
> your Fiber auth is PLOAM or LOID ?

I have no clue since this is a multi-ISP Open Access Network maintained by another big company. They won't give out any information to the ISPs and this is the only ONT provided by them. Is there a way to debug live and try to get the information with the DFP-34X-2C2?

> please get more info about your current ONU like this

It's not responding at the default address listed here (at least when PON is connected). Is there a way to debug like doing a packet trace with wirehark? Is it possible to reset and get the information before PON provision?

**Reply by Anime4000** [reply]
I can't be sure, Open Network like mine they still need PLOAM Password, any Serial Number, any MAC Address is okay, aslong PLOAM Password is correct

you, need to try by yourself,

try login into your current ONU without fiber connected (prevent OMCI provision)

most ONU can login WebGUI when fiber disconnected after power on,

simply on ONU without fiber connected, login their default IP, usually 192.168.1.1, admin, adminHW as password
